relay-server/src/services/processor.rs

```diff
@@ -1208,8 +1208,33 @@ impl EnvelopeProcessorService {
                 max_secs_in_future: Some(self.inner.config.max_secs_in_future()),
                 is_validated: false,
             };
+
+            let key_id = state
+                .project_state
+                .get_public_key_config()
+                .and_then(|key| Some(key.numeric_id?.to_string()));
+            // TODO(iker): log the missing key_id error in all internal relays
```

Comment:
When relay supports configuration to run normalization (run on pops, not processing), I'll remove this `todo`.